### DISCUSSION TITLE
Failed to configure a DataSource: 'url' attribute is not specified and no embedded datasource could be configured.

### DISCUSSION THREAD

**Discussion by xiaobeigua** [zuihou/zuihou-admin-cloud#76] [01.常见问题]
版本说明：
nacos140
mySql版本8.0.20
代码版本3.0.2
遇到的问题：
![image](https://user-images.githubusercontent.com/76942175/107306891-b2258300-6ac0-11eb-9c0b-d43c7379163f.png)
我已经按照问题都配置过了，现在不知道问题出在那里，请大佬帮忙看看
我的配置如下
![YMV{153Z) 1HTN}}~1JI5X1](https://user-images.githubusercontent.com/76942175/107307061-fadd3c00-6ac0-11eb-8d9b-2557cf4d48b0.png)
![15M`JD83SPG4 X ~X({25UV](https://user-images.githubusercontent.com/76942175/107307071-fe70c300-6ac0-11eb-8b65-0974a6172de8.png)
![ROTP{HWLW%GL~X}CCLR`5H0](https://user-images.githubusercontent.com/76942175/107307082-00d31d00-6ac1-11eb-8ad7-eedd80d8289f.png)
![RUM {8FYM Z@DBAQ TEH5G1](https://user-images.githubusercontent.com/76942175/107307086-029ce080-6ac1-11eb-8419-bf6b828d5bac.png)
![1X131H)XMW64M2 MRXFB27P](https://user-images.githubusercontent.com/76942175/107307090-04ff3a80-6ac1-11eb-8677-312d7b26b365.png)

**Comment by zuihou** [comment]
https://github.com/zuihou/lamp-cloud/discussions/49

**Comment by zuihou** [comment]
一定要注意： 项目配置文件（config-dev.properties）中填写的 `nacos.namespace` ，

一定是nacos中的`命名空间ID` ！！！

不是`命名空间名称` ！！！

![image](https://user-images.githubusercontent.com/8500410/115827452-df869980-a43e-11eb-9b9b-6ac6bbfe2bb0.png)

**Reply by 18654547573** [reply]
我也是同样问题，你说的解决方法 我都注意了，还是没解决

**Reply by 18654547573** [reply]
![image](https://user-images.githubusercontent.com/56111928/115831799-a51ffb00-a444-11eb-9065-55ec30bd4fbd.png)

**Reply by 18654547573** [reply]
![image](https://user-images.githubusercontent.com/56111928/115831923-cf71b880-a444-11eb-8fa3-ea05534823d6.png)

**Reply by aiPlusPlus6** [reply]
我也遇到同样问题后续怎么解决的？

**Reply by aiPlusPlus6** [reply]
> 我也遇到同样问题后续怎么解决的？

[已解决，采用的是docker部署nacos然后未开放端口号9848，参考(https://developer.aliyun.com/ask/686567)
